Sample Data/Custom/DSTIMAccess_CL.json

```diff
@@ -0,0 +1,60 @@
+{
+	"TenantId": "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx",
+	"SourceSystem": "RestAPI",
+	"MG": "",
+	"ManagementGroupName": "",
+	"TimeGenerated": "2021-11-18T21:58:38.0289591Z",
+	"Computer": "",
+	"RawData": "",
+	"TimeDiscovered_t": "2021-06-18T21:58:38.0289591Z",
```

Comment:
can you please remove the suffix like _s, _t etc, they will be added automatically when the data is ingested.

Reply:
@v-jayakal fixed